frontend/src/components/Layout.js

```diff
@@ -10,38 +9,16 @@ const Wrapper = styled.div`
 const Content = styled.main`
   height: 100vh;
   margin: 0 auto;
-  padding: 5em 2em;
+  padding: 7em 2em 7em 3.5em;
   width: 100%;
 `;
 
-const Footer = styled.footer`
-  background-color: ${CORAL};
-  bottom: 0;
-  height: 30px;
-  position: fixed;
-  width: 100%;
-`;
-
-const FooterContent = styled.div`
-  align-items: center;
-  display: flex;
-  height: 100%;
-  margin-left: 2em;
-`;
-
 // Main layout of the project that includes both header and navbar
 const Layout = (props) => {
```

Comment:
Should you destructure this into ({ children })?

Reply:
OMG LMFAO i actually dk how i forgot to do it. Thanks good catch